lib/shore/customers/version.rb

```diff
@@ -1,5 +1,5 @@
 module Shore
   module Customers # :nodoc:
-    VERSION = '0.2.0'
+    VERSION = '0.3.0'
```

Comment:
Bump to 1.0.0, because it makes incompatible api changes (see http://semver.org/).